docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,107 @@
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
+----
+<1> Feature options can only be used with the `oc netobserv flows` command. They cannot be used with the `oc netobserv packets` command.
+
+== Basic commands
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+| Command | Description
+| flows
+| Capture flows information. For subcommands, see the "Flow capture subcommands" table.
+| packets
+| Capture packets from a specific protocol or port pair, such as netobserv packets --filter=tcp,80. For more information about packet capture, see the "Packet capture subcommand" table.
+| cleanup
+| Remove the Network Observability CLI components.
+| version
+| Print the software version.
+| help
+| Show help.
+|===
+
+== Flows capture options
+Flow capture has mandatory commands as well as additional options, such as enabling extra features about packet drops, DNS latencies, Round-trip time, and filtering.
+.`oc netobserv flows` syntax
+[source,terminal]
+----
+$ oc netobserv flows [<feature_option>] [<command_options>]
+----
+[cols="1,1,1",options="header"]
+|===
+| Option | Description | Default
+|--enable_pktdrop|  enable packet drop                  | false
+|--enable_dns|      enable DNS tracking                 | false
+|--enable_rtt|      enable RTT tracking                 | false
+|--enable_filter|   enable flow filter                  | false
+|--log-level|       components logs                     | info
+|--max-time|        maximum capture time                | 5m
+|--max-bytes|       maximum capture bytes               | 50000000 = 50MB
+|--copy|            copy the output files locally       | prompt
+|--direction|       flow filter direction               | n/a
+|--cidr|            flow filter CIDR                    | 0.0.0.0/0
+|--protocol|        flow filter protocol                | n/a
+|--sport|           flow filter source port             | n/a
+|--dport|           flow filter destination port        | n/a
+|--port|            flow filter port                    | n/a
+|--sport_range|     flow filter source port range       | n/a
+|--dport_range|     flow filter destination port range  | n/a
+|--port_range|      flow filter port range              | n/a
+|--tcp_flags|       flow filter TCP flags               | n/a
+|--icmp_type|       ICMP type                           | n/a
+|--icmp_code|       ICMP code                           | n/a
+|--peer_ip|         peer IP                             | n/a
+|--action|          flow filter action                  | Accept
+|--interfaces|      interfaces to monitor               | n/a
+|===
+
+.Example running flow capture with options:
+[source,terminal]
+----
+$ oc netobserv flows --enable_pktdrop=true  --enable_rtt=true --enable_filter=true --action=Accept --cidr=0.0.0.0/0 --protocol=TCP --port=49051
+----
+== Packets capture options
+You can filter on port and protocol for packet capture data. 
+.`oc netobserv packets` syntax
```

Comment:
```suggestion
You can filter on port and protocol for packet capture data. 

.`oc netobserv packets` syntax
```

Reply:
Add a carriage return here to fix the rendering issues

Reply:
https://github.com/netobserv/network-observability-cli/pull/85/commits/38470dbfd5b30f5e94a68873d5f32f1e4c76e999